tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.ts

```diff
@@ -56,58 +79,83 @@ Polymer({
       type: Array,
       value: null,
     },
-    _stepBreakdownEle: {
+    _stepBreakdownItems: {
       type: Array,
-    },
-    _channelEle: {
+      value: () => [
+        {key: 'highFlopsComputeUs', label: 'High flops compute'},
+        {key: 'lowFlopsComputeUs', label: 'Low flops compute'},
+        {key: 'hostInfeedDurationUs', label: 'Infeed'},
+        {key: 'hostOutfeedDurationUs', label: 'Outfeed'},
+        {key: 'crsDurationUs', label: 'All reduce'},
+        {key: 'sendDurationUs', label: 'Send'},
+        {key: 'recvDurationUs', label: 'Recv'},
+      ],
+    },
+    _channelItem: {
       type: Array,
+      value: () => [{key: 'durationUs', label: 'Duration (us)'}],
     },
-    _allReduceEle: {
+    _allReduceItem: {
       type: Array,
+      value: () => [{key: 'durationUs', label: 'Duration (us)'}],
     },
     _stepBreakdownFunc: {
       type: Object,
+      value: (d) => {return '(' + d.chipId + ',' + d.nodeId + ')';},
```

Comment:
I believe this is incorrect. The right thing:
```ts
_stepBreakdownFunc: {
      type: Function
      value:  () => (d: TYPEME) => `(${d.chipId},${d.nodeId})`,
},
```

Seems like the same mistake is made below as well.

Reply:
Done.